config/database.yml

```diff
@@ -4,22 +4,28 @@
 #   Ensure the SQLite 3 gem is defined in your Gemfile
 #   gem 'sqlite3'
 #
-default: &default
-  adapter: sqlite3
-  pool: 5
-  timeout: 5000
-
 development:
-  <<: *default
-  database: db/development.sqlite3
-
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
+  adapter: mysql
+  encoding: utf8
+  reconnect: false
+  database: blogpost
+  pool: 5
+  username: root
+  password: **pass**
```

Comment:
put username and password like such:
[](http://stackoverflow.com/questions/17151324/securely-providing-the-database-password-in-a-rails-app)